### ISSUE TITLE
Allow Enter key to mimic mouse double click actions in sidebar

### ISSUE THREAD

**Issue by audio2000** [davy7125/polyphone#233]
I apologize if I am overlooking something.
Suggesting that maybe the “Enter” key can be programmed to mimic LMB double click actions in the sidebar.

`CURRENT LMB DOUBLE CLICK BEHAVIOR`
-Drop down based items = Expands/collapses items
-"Instruments" items = Opens that item in its “Samples” location
-"Presets" items = Opens that item in its “Instruments” location

I was wondering if it would be considered to allow the “Enter” key to perform the exact same actions in the sidebar, if would not cause any conflicts.  The drop down one mentioned I know can be covered by using "Right Arrow" but just mentioned to keep consistant with double click action of mouse.

Thank You

**Comment by davy7125** [comment]
This is not a standard behavior.
Instead, please use the right / left arrows for expanding / collapsing elements.

**Comment by audio2000** [comment]
@davy7125 
Okay, I agree about the expand / collapse elements, but
What about "Enter" to open the items though?  You do not like that?
Seems like standard behavior, similar to how double click is used to open items.

**Comment by audio2000** [comment]
Oh, no problem, just thank you for being willing to add any of this.
Yeah I had noticed that some applications use this and some do not.
What really gets me are the programs that do not offer the right and left arrows to expand and collapse (which is standard expected behavior) and only use the Enter key to achieve this! which I have seen :)
Again I was just mentioning this for consistency to match how the mouse functions, but the main one was the opening of items which is great to hear that this was added.
Thank you again
